build-study.sh

```diff
@@ -27,7 +28,6 @@ docker_push() {
 
 VERSION=$1; shift
 ENV=$1; shift
-DIR=$1; shift
```

Comment:
Getting rid of DIR from command line. Script always executed from same place and we know what directory to check.

Reply:
This is not always the case- the Jenkins build invokes `build-study.sh` with a value of `/app` for DIR. Have you make sure this still works in Jenkins, or is updated if needed?

Reply:
Actually, on closer inspection I think we should be good as is. The point of this PR is to do away for the need for Angular docker deployment. 
So all the Docker called by Jenkins to build Angular and then deploy the Docker image to GCP will go bye-bye.
The script that I am intending to execute from Jenkins to replace the Docker deployment is `deploy-to-appengine.sh` which is also part of this PR.
This script will generate the config file, build the Angular locally and then call the GCP utility to upload to Google App Engine. No Docker rigamarole required.

When this PR is approved, my intention is to merge and immediately try to get it integrated into the Jenkins build. Main thing will be to have Jenkins execute the `deploy-to-appengin.sh` script.

I also **MIGHT** go back and remove all the Docker related stuff. I say might because perhaps there is still some utility for it locally. Doubt it, but possible.

Reply:
I added `DIR` back in and tweaked jenkins job so it sets `ENVIRONMENT` _and_ `ENV`, thus enabling concurrent builds via jenkins and  circle.

<img width="1313" alt="Screen Shot 2020-02-12 at 6 24 13 PM" src="https://user-images.githubusercontent.com/1653048/74386856-0a9a4580-4dc5-11ea-8e61-a79c75a98a66.png">